style.css

```diff
@@ -0,0 +1,96 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+.textarea {
+  width: 100%;
+  height: 50%;
+  position: absolute;
+}
```

Comment:
I tend to think that `resize: none` here will help us to save the layout as intended